charmcraft/helptexts.py

```diff
@@ -73,12 +73,21 @@ def init(self, appname, general_summary, command_groups):
         self.general_summary = general_summary
         self.command_groups = command_groups
 
-    def get_usage_message(self, fullcommand, error_message):
+    def get_usage_message(self, extra_command, error_message):
```

Comment:
Would `subcommand` be a better name in this case?

Reply:
It could be just `command`, as in reality currently charmcraft just passes the command (e.g. `pack`) or nothing, when the usage help requested is "general".

I could probably make it `""` by default so it's optional.

What do you think?

Reply:
I went forward with this.